docs/assets/zdm-config-reference.yml

```diff
@@ -0,0 +1,170 @@
+# This variable determines which cluster is currently considered the primary cluster.
```

Comment:
Add a paragraph at the top saying that required settings are uncommented and all optional ones are commented.